rs/rosetta-api/icp/client/src/lib.rs

```diff
@@ -1104,6 +1136,33 @@ impl RosettaClient {
         .await
     }
 
+    pub async fn register_vote<T>(
```

Comment:
Would be great to also have a description of the function, as we want to release the client to crates.io.

Reply:
Done.